velox/experimental/exec/OffProcessExpressionEval.cpp

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "velox/experimental/exec/OffProcessExpressionEval.h"
+
+namespace facebook::velox::exec {
+
+// Maximum buffer size in bytes for stream groups before they are
+// flushed.
+constexpr int32_t kMaxStreamGroupSizeBytes{50000};
```

Comment:
placeholder value for now. Benchmarks and experiments will come in the next PRs.